### PR TITLE
Feature/configure app

### LINKED ISSUE #2: configure app
As a dev,
* I would like to have a component structure set up to start adding api calls, adding logic for state, and adding styles.

### PR DESCRIPTION
**Pull request by cameronRomo**
### What’s this PR do?

* Configures react app and adds fundamental components

### Where should the reviewer start?

* `src/App/App.js` line 1
*  `src/Card/Card.js` line 1
* `src/Experiences/Experiences.js`  line 1 (set up for next branch)
* `src/NavBar/NavBar.js` line 1
*  `src/Strains/Strains.js` line 1

### How should this be manually tested?

* Pull down and run `npm start`, then select a strain and see pertinent data render on the page

### Any background context you want to provide?

* React with hooks is the method for keeping state

### Screenshots (if appropriate):

Screen Shot 2021-01-15 at 7 43 21 PM

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY = HEVGp9F
```
Comment (brabbuss) [inline .env:1]: Smaht

File: src/App/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { Strains } from '../Strains/Strains';
+// import Experiences from '../Experiences/Experiences';
+import { Route, Switch, Redirect } from 'react-router-dom';
+
+export const App = () => {
+  return (
+    <section>
+      <NavBar />
+      <Switch>
+        <Redirect exact path="/" to="/strains" component={Strains} />
```
Comment (brabbuss) [inline src/App/App.js:13]: what's going on here? if you have just one endpoint '/strains' then just use that one route. if you have 2, order them from most specific to least, and you don't need an exact at that point

File: src/Experiences/Experiences.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Experiences.css';
+
+export const Experiences = ({ strain }) => {
+  return (
+    <section>
+
```
Comment (brabbuss) [inline src/Experiences/Experiences.js:7]: I find it helpful to throw some temp placeholder text in empty components that actually render - so y ou know they're rendering properly

File: src/index.js [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { App } from './App/App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <BrowserRouter>
```
Comment (brabbuss) [inline src/index.js:9]: don't forget your base pathj

**Review by brabbuss — APPROVED** [review]
Approved but see comments

File: .env (hunk 1 above)
Comment (Codo-Baggins) [reply .env:1]: Yeah make sure you put this in the .gitignore file! Don't want to merge this for others to see.

File: src/Card/Card.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        <h5>Effects:</h5>
+        <h6>Positve:</h6>
+        {props.strain.effects.positive.map(flavor => (<ul>{flavor}</ul>) )}
+        <h6>Negative:</h6>
+        {props.strain.effects.negative.map(flavor => (<ul>{flavor}</ul>) )}
+        <h6>Medical:</h6>
+        {props.strain.effects.medical.map(flavor => (<ul>{flavor}</ul>) )}
+        <img src={leaf}/>
+      </div>
+    </article>
```
Comment (Codo-Baggins) [inline src/Card/Card.js]: I think this makes more sense semantically to be a section.  See the documentation on article for more specifics or reach out with questions!

File: src/Experiences/Experiences.js (hunk 3 above)
Comment (Codo-Baggins) [reply src/Experiences/Experiences.js:7]: Im in the same boat here^

**Review by Codo-Baggins — CHANGES_REQUESTED** [review]
Overall, your project looks really good so far! Just add that .env file to your .gitignore and then I will merge!

File: .gitignore [hunk 6]
```diff
@@ -1,4 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+# API Key
+.env
```
Comment (cameronRomo) [inline .gitignore:3]: Like this?

File: src/Card/Card.js (hunk 5 above)
Comment (cameronRomo) [reply src/Card/Card.js]: I see what you're saying. I went ahead and changed article to section.

File: src/Experiences/Experiences.js (hunk 3 above)
Comment (cameronRomo) [reply src/Experiences/Experiences.js:7]: Good call! I think I just got caught second-guessing what I wanted to do with Experiences. My thoughts are to render all the saved strains, and potentially use local storage.

**Pull request merged**